### ISSUE TITLE
Add an image graphic to map body

### ISSUE THREAD

**Issue by MesquiteGIS** [cmv/cmv-app#436]
I'd like to add a graphic to the map body of my viewer. In my case, just a simplified legend (circled in red). Should be a fairly simple task i would think, but I'm not sure how to do this. Any help would be much appreciated.

Thank you
![sample](https://cloud.githubusercontent.com/assets/7807619/8789081/47b44e88-2ef4-11e5-8419-d22992ba5fa2.PNG)

**Comment by tmcgee** [comment]
A few quick thoughts off the top of my head assuming it is a static image:

1: Most obvious place is to add an `image` tag into is in the [`mapOverlay` template:](https://github.com/cmv/cmv-app/blob/master/viewer/js/viewer/templates/mapOverlay.html)

2. If you wanted to make a widget, you could do a simple widget of type `domNode` with the image tag in the `content` parameter of the widget's options. This could probably be done completely in the `viewer.js` config file without the need for separate widget files (HTML/JS/CSS).

**Comment by MesquiteGIS** [comment]
That actually helps a lot Tim. Thank so much. I'll give that a try.

**Comment by MesquiteGIS** [comment]
getting close.. Think I may need to re-size my image. LOL
![big legend](https://cloud.githubusercontent.com/assets/7807619/8789837/67a7648a-2efb-11e5-8ef2-afaa09da2039.PNG)

**Comment by MesquiteGIS** [comment]
As a follow up... if it helps anyone else. Tim's suggestion work perfect. To add a static image to my viewer;
First, I added an image tag into  the mapOverlay template:

<div style="position:absolute;top:20px;left:60px;z-index:40;">
    <div id="legendImage">
		<img  src="images/SurfaceConditionIndex.png"  />
    </div>
</div>

**Comment by MesquiteGIS** [comment]
As a follow up... if it helps anyone else. Tim's suggestion work perfect. To add a static image to my viewer;
First, I added an image tag into  the mapOverlay template:

<div style="position:absolute;top:20px;left:60px;z-index:40;">
    <div id="legendImage">
		<img  src="images/SurfaceConditionIndex.png"  />
    </div>
</div>

Then I added the "legendImage" as a widget in the viewer.js configuration file;

legendImage: {
                include: true,
                id: 'legendImage',
                type: 'domNode',
                path: 'images/SurfaceConditionIndex.png',
                
            },


The end result was a nice static legend image.
![legend](https://cloud.githubusercontent.com/assets/7807619/8790104/bbde853a-2efe-11e5-85fd-53d08c3bf2a7.PNG)
